sympy/parsing/mathematica.py

```diff
@@ -1135,8 +1140,10 @@ def converter(expr):
         "Equal": Equality,
         "Or": Or,
         "And": And,
-
+        "Not": Not,
         "Function": _parse_Function,
+        "Factorial": factorial,
+        "Factorial2": lambda x: factorial(x) * factorial(x - 1),
```

Comment:
What is factorial2 here doing? Is this tested?

This formula does not look correct. The associated sympy function is called factorial2 and is not equivalent to `factorial(x) * factorial(x - 1)`.